test-failure-policy

```diff
@@ -66,6 +68,12 @@ def main():
             print("due to failure of test harness or framework")
             return 1
 
+        if not number and opts.all:
```

Comment:
this would be better as an "else" branch after line 63, to keep the "known issue" stuff together. It's confusing and also inefficient to have the "checkRetry" thing in between.